locksmith/src/routes/v2/metadata.ts

```diff
@@ -0,0 +1,53 @@
+import express from 'express'
+import { MetadataController } from '../../controllers/v2/metadataController'
+import { authenticatedMiddleware } from '../../utils/jwt'
+
+const router = express.Router({ mergeParams: true })
+
+const metadataController = new MetadataController()
+
+router.get('/:network/locks/:lockAddress', (req, res) =>
+  metadataController.getLockMetadata(req, res)
+)
+
+router.put(
+  '/:network/locks/:lockAddress',
+  authenticatedMiddleware,
```

Comment:
Can you confirm this will only let a lockManager do this?

Reply:
Yes. The check is in the controller method.